app/views_rails_6_1/layouts/_head_tag_content.html.erb

```diff
@@ -0,0 +1,46 @@
+<%# OVERRIDE Hyrax 4.0 to allow for dual boot %>
+<%= csrf_meta_tag %>
+<meta charset="utf-8" />
+<%# Only display meta tag, which enables creation of the ActionCable
+consumer, when realtime notifications are enabled and the user is
+signed in %>
+<% if Hyrax.config.realtime_notifications? && signed_in? %>
+    <%= tag :meta, name: 'current-user', data: { user_key: current_user.user_key } %>
+<% end %>
+<!-- added for use on small devices like phones -->
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="resourcesync" href="<%= hyrax.capability_list_url %>" />
+
+<!-- Twitter card metadata -->
+<%= yield :twitter_meta %>
+<!-- Google Scholar metadata -->
+<%= yield :gscholar_meta %>
+
+<title><%= content_for?(:page_title) ? yield(:page_title) : default_page_title %></title>
+
+<%# OVERRIDE to allow for dual boot start %>
```

Comment:
I don't think we need this file either. Does it get hit? It's identical to the non rails_6_1 directory

Reply:
Same as above ⬆️